css/index.css

```diff
@@ -112,6 +142,36 @@ p {
     justify-content: center;
 }
 
+.boxes .box:nth-child(1){
```

Comment:
Great job using CSS selectors rather than creating a brand new class for every box